packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/SourceNetworkBox.tsx

```diff
@@ -95,6 +95,25 @@ export function SourceNetworkBox({
     }
   }, [maxAmount2, setAmount2])
 
+  const handleAmountChange: ChangeEventHandler<HTMLInputElement> = useCallback(
+    e => setAmount(e.target.value),
+    [setAmount]
+  )
+  const handleAmount2Change: ChangeEventHandler<HTMLInputElement> = useCallback(
+    e => {
+      setAmount2(e.target.value)
+    },
+    [setAmount2]
+  )
+
+  const tokenButtonOptions = useMemo(
```

Comment:
Could we specify that's it's for `amount2`, something like `tokenButtonOptionsAmount2` or `tokenButtonOptionsInput2` whatever makes sense, open to other suggestions